multihost_test/Bugzillas/test_regression.py

```diff
@@ -170,3 +170,29 @@ def test_bz_1206273(self, multihost):
         assert cmd1.returncode == 0
         assert cmd3.returncode == 0
         assert user in cmd2.stdout_text
+
+    def test_bz_1089666(self, multihost):
+        """
+        :title: 'usermod -m -d ' should warn that it did not create
+            destination(new)home directory if source home directory does not exist
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1089666
+        :id: 8cb0f63a-b80d-11ed-a4d5-845cf3eff344
+        :steps:
+          1. Create a new user on the system using the useradd command
+          2. Recursively removes the home directory of the user using rm -vfr
+          3. Modify the user's home directory using usermod, and
+            redirects any error messages to a file called /tmp/anuj.
+          4. Deletes the user and all associated files using userdel -rf
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+        """
+        client = multihost.client[0]
+        user = "bz1089666_user"
+        client.run_command(f"useradd {user}")
+        client.run_command("rm -vfr /home/bz1089666_user")
+        client.run_command(f"usermod -m -d '/home/bz1089666_user_2' {user} &> /tmp/anuj")
+        client.run_command(f"userdel -rf {user}")
+        assert 'Move cannot be completed.' in client.run_command("cat /tmp/anuj").stdout_text
```

Comment:
You should also check that the new directory `/home/bz1089666_user_2` doesn't exist.